src/main/kotlin/io/github/gradlenexus/publishplugin/internal/StagingRepositoryTransitioner.kt

```diff
@@ -59,7 +59,7 @@ class StagingRepositoryTransitioner(val nexusClient: NexusClient, val retrier: A
     private fun assertRepositoryInDesiredState(repository: StagingRepository, vararg desiredStates: StagingRepository.State) {
         if (repository.state !in desiredStates) {
             throw RepositoryTransitionException(
-                "Staging repository is not in desired state ${desiredStates.contentToString()}: $repository. It is unexpected. Please check " +
+                "Staging repository is not in desired state ${desiredStates.contentDeepToString()}: $repository. It is unexpected. Please check " +
```

Comment:
Is this a backward compatible change? When was it introduced (before 6.2?)?

Reply:
[`contentToString`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/content-to-string.html) seems actually Kotlin 1.4, and we're targeting Kotlin 1.3. Not sure how it was compiling before. [`contentDeepToString`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/content-deep-to-string.html) is Kotlin 1.1

It's very likely some kind of inference / overload resolution change.